app/database/models/server/file.ts

```diff
@@ -54,4 +54,22 @@ export default class FileModel extends Model {
 
     /** post : The related Post record for this file */
     @immutableRelation(POST, 'post_id') post!: Relation<PostModel>;
+
+    toFileInfo = (authorId: string): FileInfo => ({
+        id: this.id,
+        user_id: authorId,
+        post_id: this.postId,
+        create_at: 0,
+        delete_at: 0,
+        update_at: 0,
```

Comment:
We don't need this information?

Reply:
we don't care that is why is not being stored. I'll make it optional for FileInfo